### PR TITLE
added the 60fps patch to GTA San Andreas 1.03 US

### PR DESCRIPTION
**Pull request by bizarf**
I added the 60fps patch to Grand Theft Auto: San Andreas (SLUS-20946GH) / Ver 1.03 (CRC 399A49CA) that LilNizzy from RetroAchievements created
https://retroachievements.org/viewtopic.php?t=16674&c=192090#192090

### CODE REVIEW

File: patches/SLUS-20946_399A49CA.pnach [hunk 1]
```diff
@@ -25,3 +25,8 @@ description=Removes the radiosity filter which causes a ghosting effect on the p
 
 //Remove Radiosity Filter
 patch=1,EE,20519FD8,extended,00000000
+
+[60 FPS Patch]
```
Comment (pgert) [inline patches/SLUS-20946_399A49CA.pnach:29]: It's usually called "[60 FPS]", not "[60 FPS Patch]".

File: patches/SLUS-20946_399A49CA.pnach [hunk 2]
```diff
@@ -25,3 +25,8 @@ description=Removes the radiosity filter which causes a ghosting effect on the p
...
+author=LilNizzy, Alexkander_
```
Comment (pgert) [inline patches/SLUS-20946_399A49CA.pnach:30]: author=asasega

https://forums.pcsx2.net/Thread-60-fps-codes?pid=551178#pid551178
The post by asasega is from 2017.
The post by LilNizzy is from 2023.

File: patches/SLUS-20946_399A49CA.pnach [hunk 3]
```diff
@@ -25,3 +25,8 @@ description=Removes the radiosity filter which causes a ghosting effect on the p
...
+description=Make sure you set your EE Cycle Rate to 130-180%
```
Comment (pgert) [inline patches/SLUS-20946_399A49CA.pnach:31]: "comment", not "description".
The statement doesn't describe anything, only advices.

**Comment by bizarf** [comment]
I'm so sorry about those dumb mistakes. I've corrected them.

**Pull request merged**